dss/util/auth/auth0.py

```diff
@@ -24,21 +24,41 @@ def security_flow(self, *args, **kwargs):
         kwargs of this method, and used to call the correct method.
         """
         #  TODO add some type of jwt inspection
-        requested_method = kwargs.get('auth_method').lower()
-        if requested_method is None or requested_method not in self.valid_methods.keys():
-            err = f'Unable to locate auth_method {requested_method} for request, valid methods are: '
-            err += f'{", ".join(self.vaid_methods)}'
-            raise DSSException(500, err)
+
+        # Get name of method to use
+        if 'auth_method' in kwargs:
```

Comment:
this same function block from the other PR! >:0

Reply:
That one was for `security_groups`, which is at `args[0]` for Fusillade. This one is for `auth_method`, which is at `args[0]` for Auth0.

I think the cleanest approach is to do away with any positional arguments, and require the user to specify things by keyword.